.github/workflows/jekyll-gh-pages-avatar.yml

```diff
@@ -30,21 +30,46 @@ jobs:
         uses: actions/checkout@v3
         with:
           path: main
+
       - name: Clone autils repo
         uses: actions/checkout@v3
         with:
           repository: avocado-framework/autils
           path: resources/autils
           ref: main
-      - name: Copy autils metadate to _data dir
+
+      - name: List files in autils directory
+        run: ls -R resources/autils/metadata/autils/
+
+      - name: Copy autils metadata to _data dir
         run: cp -r resources/autils/metadata/autils/* main/_data/autils/
+
+      - name: Copy YAML files from autils
+        run: cp -r resources/autils/metadata/autils/**/*.yml main/_data/autils/
```

Comment:
Have you test it out in the workflow? I am not sure what are the shell capabilities in the GitHub workflow, therefore I am asking. If it is working, I am fine with it.

Reply:
Tested in my fork and workflow, build is still completing and everything still working on website :)

Reply:
ok great